src/views/DissolutionEntity/DissolutionEntity.vue

```diff
@@ -0,0 +1,447 @@
+  }
+
+  /** Handler for Valid change event. */
+  private onIsCertified (val: boolean): void {
+    this.setCertifyState(
+      {
+        valid: val,
+        certifiedBy: this.getCertifyState.certifiedBy
+      }
+    )
+  }
+
+  /** Handler for CertifiedBy change event. */
+  private onCertifiedBy (val: string): void {
+    this.setCertifyState(
+      {
+        valid: this.getCertifyState.valid,
+        certifiedBy: val
+      }
+    )
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+
+#dissolution-review-confirm {
+  /* Set "header-counter" to 0 */
+  counter-reset: header-counter;
+}
+
+h2::before {
+  /* Increment "header-counter" by 1 */
+  counter-increment: header-counter;
+  content: counter(header-counter) '. ';
+}
+
+#dissolution-summary .v-divider {
+  background-color: $gray1;
+}
+
+::v-deep .section-container {
+  padding: 1.5rem 1.5rem;
+}
+
+::v-deep .review-header {
+  display: flex; // to align icons
+  background-color: $BCgovBlue5O;
+  padding: 1.25rem;
+  color: $gray9;
+}
+
+.v-icon.mdi-information-outline {
+  margin-top: -2px;
+  margin-right: 4px;
+}
+
+.container {
+  padding: 2rem;
+  background-color: $gray1;
+
+  &.invalid {
+    border-left: 3px solid $BCgovInputError;
+    padding-left: calc(2rem - 3px);
+
+    label {
+      color: $BCgovInputError;
+    }
+  }
+}
+
+.inner-col-1 {
+  // adjustment to make this inner container column the same width as the outer columns
+  // ie, decrease width by 1/2 container margin + padding
+  flex: 0 0 calc(25% - 1rem);
+}
+
+.inner-col-2 {
+  // adjustment to make this inner container column the same width as the outer columns
+  // ie, increase width by 1/2 container margin + padding
+  flex: 0 0 calc(75% + 1rem);
+  max-width: calc(75% + 1rem);
+}
+
+#effective-date-text {
+  color: $gray7;
+}
+
+// FUTURE: this should be under court-order-poa-section below
+::v-deep #court-order {
+  .row {
+    .col-9 {
+      padding-left: 1rem !important;
+    }
+  }
+
+  #court-order-label, #poa-label {
+    font-size: $px-16;
+    font-weight: bold;
+    color: $gray9;
+  }
+}
+
+.upload-success-message {
+  color: $gray7;
+
+  .success-chk {
+    font-size: $px-24;
+    color: $app-dk-green;
+  }
+
+  #file-name-col {
+    margin-top: 2px;
+    margin-left: -46px;
+  }
+}
+
+// styles common to the sections
+::v-deep #document-delivery-section,
+::v-deep #folio-number-section,
+::v-deep #certify-section,
+::v-deep #court-order-poa-section,
+::v-deep #staff-payment-section {
+  .v-card {
+    padding: 1.5rem 1.25rem !important;
+  }
+
+  .row {
+    padding: 0.75rem 0;
+
+    .col-3 {
+      font-size: $px-16;
+      color: $gray9;
+      padding: 0 0 0 0.75rem !important;
+    }
+
+    .col-9 {
+      padding: 0 0.5rem 0 0 !important;
+    }
+  }
+}
+
+// styles specific to certify section
+::v-deep #certify-section {
+  .v-card {
+    margin-top: 0 !important;
+  }
```

Comment:
Same as above

Reply:
Yes, mostly we how use v-cards with no extra CSS (ie, `<v-card flat>`) (although probably some top padding for whitespace).